testing/tests/DevExpress.ui.widgets/treeViewParts/searching.js

```diff
@@ -497,3 +497,38 @@ QUnit.test('apply search after searchTimeout', function(assert) {
     assert.equal($items.length, 4, 'filter was applied after timeout');
     this.clock.restore();
 });
+
+['none', 'selectAll', 'normal'].forEach(selectionMode => {
+    QUnit.test(`focusIn -> search -> focusIn -> clearSearch -> focusIn. selectionMode: ${selectionMode}`, function(assert) {
+        const treeView = initTree({
+            items: [{ id: 1, text: 'item1' }, { id: 2, text: 'item2' }],
+            searchEnabled: true
+        }).dxTreeView('instance');
+
+        const isNodeFocused = (id) => treeView.$element().find(`[data-item-id="${id}"]`).hasClass('dx-state-focused');
+        const triggerFocus = () => treeView._focusInHandler({});
+
+        assert.equal(isNodeFocused(1), false, 'item1 is not focused after initialization');
+        assert.equal(isNodeFocused(2), false, 'item2 is not focused after initialization');
```

Comment:
Will we check 'focusedElement'? It's an independent reference to a focused element.
assert.equal($(this.instance.option('focusedElement')).text(), 'item3'); ?